packages/dd-trace/src/service-naming/index.js

```diff
@@ -19,6 +19,9 @@ class SchemaManager {
   }
 
   serviceName (type, kind, plugin, ...serviceNameArgs) {
+    if (this.config.traceRemoveIntegrationServiceNamesEnabled && this.version === 'v0') {
```

Comment:
Isn't the whole point of service naming to avoid this type of thing? Why is it not 2 different versions at that point?

Reply:
It's been decided to do it this way across tracers to allow customers to streamline service names without requiring to opt-in to naming versions in general.